src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,39 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+class FigureSupplier {
+    public static Figure getRandomFigure() {
+        Random random = new Random();
+        Colors color = ColorSupplier.getRandomColor();
+        int figureType = random.nextInt(5);
+        switch (figureType) {
+            case 0:
+                return new Square(color,
+                        (random.nextInt(20)));
```

Comment:
```suggestion
                return getRandomSquare();
```
add private methods to get each figure - this would make your code easier to read